### PR TITLE
HDDS-6190. Cleanup unnecessary datanode id path checks.

### PR DESCRIPTION
**Pull request by guihecheng**
## What changes were proposed in this pull request?

Cleanup unnecessary datanode id path checks.
More details on the JIRA below.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-6190

## How was this patch tested?

CI: https://github.com/guihecheng/ozone/actions/runs/1706899399

### CODE REVIEW

**Review by adoroszlai — COMMENTED** [review]
Thanks @guihecheng for the patch.  Functionality looks good to me.  Can you please add a unit test in `TestHddsServerUtil` to verify the behavior of `getDatanodeIdFilePath` (falls back to meta dir if null or empty, uses specific value if set)?

**Comment by guihecheng** [comment]
Hi @adoroszlai , sorry to be a bit late, updated with a new ut, thanks~

File: hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestHddsServerUtils.java [hunk 1]
```diff
@@ -31,11 +31,9 @@
 import org.apache.hadoop.test.PathUtils;
 
 import org.apache.commons.io.FileUtils;
-import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_CLIENT_ADDRESS_KEY;
-import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_DATANODE_ADDRESS_KEY;
-import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_DATANODE_PORT_DEFAULT;
-import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_NAMES;
-import static org.apache.hadoop.hdds.scm.ScmConfigKeys.OZONE_SCM_STALENODE_INTERVAL;
+
+import static org.apache.hadoop.hdds.scm.ScmConfigKeys.*;
```
Comment (adoroszlai) [inline hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/TestHddsServerUtils.java:35]: Nit: please avoid `*` import.

**Review by adoroszlai — COMMENTED** [review]
Thanks @guihecheng for adding the test.  Looks good, except the import.

**Review by adoroszlai — APPROVED** [review]
Thanks @guihecheng for updating the patch.

**Pull request merged**